File: sample/src/main/java/tech/thdev/androidrecyclerview/ui/custom_scroll/basic/BasicCustomScrollActivity.java
package tech.thdev.androidrecyclerview.ui.custom_scroll.basic;

import android.os.Bundle;
import android.support.annotation.Nullable;

import tech.thdev.androidrecyclerview.R;
import tech.thdev.base.util.ActivityUtilKt;
import tech.thdev.base.view.BaseActivity;

/**
 * Created by Tae-hwan on 18/10/2016.
 */

public class BasicCustomScrollActivity extends BaseActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_basic_custom_scroll);

        ActivityUtilKt.replaceContentFragment(this, R.id.frame_layout, BasicCustomScrollFragment.getInstance());
    }
}

File: sample/src/main/java/tech/thdev/androidrecyclerview/ui/custom_scroll/basic/BasicCustomScrollActivity.kt
package tech.thdev.androidrecyclerview.ui.custom_scroll.basic

import android.os.Bundle
import tech.thdev.androidrecyclerview.R
import tech.thdev.androidrecyclerview.databinding.ActivityBasicCustomScrollBinding
import tech.thdev.base.ui.BaseActivity
import tech.thdev.base.util.replaceContentFragment

/**
 * Created by Tae-hwan on 18/10/2016.
 */
class BasicCustomScrollActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding = ActivityBasicCustomScrollBinding.inflate(layoutInflater)
        setContentView(binding.root)

        replaceContentFragment(R.id.frame_layout, BasicCustomScrollFragment.newInstance())
    }
}